<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockWidget</class>
 <widget class="QWidget" name="BlockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string>Add...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="chkInches">
     <property name="text">
      <string>Length in Inches</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="chlCentimeters">
     <property name="text">
      <string>Length in Centimeters</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="JTable" name="tableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JTable</class>
   <extends>QTableView</extends>
   <header location="global">jtable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
